src/externals/crypto/aes.ts

```diff
@@ -75,13 +71,13 @@ export class AesService {
 
   decrypt(encdata, folderId) {
     // base64 decoding
-    const bData = Buffer.from(encdata, 'base64');
+    const bData: Buffer = Buffer.from(encdata, 'base64');
```

Comment:
These types are trivially inferred by TypeScript, there is no need to add them. You should add them when the result of a function is any, unknown, or you are using concrete implementations but you want to force the usage of the interface contract. Example: 

```
interface FilesRepository {
   findAll(): Promise<...>
}

class MongoDBFilesRepository implements FilesRepository {
   ....
   methodThatIsNotInTheInterface()...
}

class MariaDBFilesRepository implements FilesRepository {
  ....
}

// here you want to force the interface usage, so the code will 
// be compatible instantly if it is replaced by MariaDB in any situation
const repository: FilesRepository = new MongoDBRepository();
repository.methodThatIsNotInTheInterface() // TypeScript won't allow this as it can not ensure that this a MongoDBRepository.

```